yarn.lock

```diff
@@ -1231,6 +1231,11 @@
   version "7946.0.1"
```

Comment:
Strange that it left the old geojson types, can you delete this block and make line 1234:

```
"@types/geojson@*", "@types/geojson@7946.0.8":
```